Remotion/ObjectBinding/UnitTests/BindableObject/ReferencePropertyTests/Delete.cs

```diff
@@ -48,66 +46,59 @@ public override void SetUp ()
     [Test]
     public void Delete_WithDeleteSupported ()
     {
-      var stubBusinessObject = _mockRepository.Stub<IBusinessObject>();
-      var mockService = _mockRepository.StrictMock<IDeleteObjectServiceOnProperty>();
+      var stubBusinessObject = new Mock<IBusinessObject>();
+      var mockService = new Mock<IDeleteObjectServiceOnProperty> (MockBehavior.Strict);
       IBusinessObjectReferenceProperty property = DeleteProperty ("DeleteObjectServiceFromPropertyDeclaration");
-      var value = _mockRepository.Stub<IBusinessObject>();
+      var value = new Mock<IBusinessObject>();
 
-      using (_mockRepository.Ordered())
-      {
-        mockService.Expect (mock => mock.SupportsProperty (property)).Return (true);
-        mockService.Expect (mock => mock.Delete (stubBusinessObject, property, value));
-      }
-      _mockRepository.ReplayAll();
+      var sequence = new MockSequence();
+      mockService.InSequence (sequence).Setup (mock => mock.SupportsProperty (property)).Returns (true).Verifiable();
+      mockService.InSequence (sequence).Setup (mock => mock.Delete (stubBusinessObject.Object, property, value.Object)).Verifiable();
 
-      _bindableObjectProviderForDeclaringType.AddService (mockService);
-      property.Delete (stubBusinessObject, value);
+      _bindableObjectProviderForDeclaringType.AddService (mockService.Object);
+      property.Delete (stubBusinessObject.Object, value.Object);
 
-      _mockRepository.VerifyAll();
+      mockService.Verify();
     }
 
     [Test]
     public void Create_WithDeleteSupportedAndReferencingObjectNull ()
     {
-      var mockService = _mockRepository.StrictMock<IDeleteObjectServiceOnType> ();
+      var mockService = new Mock<IDeleteObjectServiceOnType> (MockBehavior.Strict);
       var property = DeleteProperty ("DeleteObjectServiceFromPropertyType");
-      var value = _mockRepository.Stub<IBusinessObject> ();
+      var value = new Mock<IBusinessObject>();
 
-      using (_mockRepository.Ordered ())
-      {
-        mockService.Expect (mock => mock.SupportsProperty (property)).Return (true);
-        mockService.Expect (mock => mock.Delete (null, property, value));
-      }
-      _mockRepository.ReplayAll ();
+      var sequence = new MockSequence();
+      mockService.InSequence (sequence).Setup (mock => mock.SupportsProperty (property)).Returns (true).Verifiable();
+      mockService.InSequence (sequence).Setup (mock => mock.Delete (null, property, value.Object)).Verifiable();
```

Comment:
Is the but with the missing InSequence statements already fixed in the current tool version?

Reply:
`InSequence` should no longer cause a problem